external/avro/src/main/scala/org/apache/spark/sql/avro/AvroFileFormat.scala

```diff
@@ -133,8 +134,7 @@ private[avro] class AvroFileFormat extends FileFormat with DataSourceRegister {
         job.getConfiguration.set(AvroJob.CONF_OUTPUT_CODEC, DataFileConstants.SNAPPY_CODEC)
 
       case "deflate" =>
-        val deflateLevel = spark.conf.get(
-          AVRO_DEFLATE_LEVEL, Deflater.DEFAULT_COMPRESSION.toString).toInt
```

Comment:
I changed it because I didn't have any ideas what `-1` means. Is it closer to best compression `9` or fast compression `1`? Probably the level is eventually set in `zlib` in which `-1` means `6`. @gengliangwang From your point of view, `-1` means better or faster compression?

Reply:
If the compression algorithm changes in the feature, the default value may not be 6. But if we use `DEFAULT_COMPRESSION`, it will still be equivalent to the new default value. So I suggest we still use `Deflater.DEFAULT_COMPRESSION` instead of any specific number.